CONTRIBUTING.md

```diff
@@ -32,10 +33,134 @@ Contributing to Cube Cobra does not entitle any contributor to compensation of a
 
 Issues are tagged with 'good first issue' if we think it's a good beginner task to tackle. Complete a few small changes to become familar with the codebase before diving into a huge feature.
 
-# Getting started
-### How to submit a contribution.
-
-The master branch is the branch that contains live code. The development branch is the branch with changes that are pending for the next update. First, assign yourself to an issue, and create a fork of the code. When you are finished with the feature, create a pull request back into the development branch. An admin will review your code and merge if it is accepted. Expect some comments and feedback.
+# How we collaborate - asynchronous communication
```

Comment:
This section is good to have, but I wonder whether it's the first thing we want to tell new contributors. It also might be a little duplicative of the "How we use Discord" section below.

Reply:
I put this section first because I feel that an overview of the philosophy informing the implementation details is almost more important than the details themselves. Since we can't cover every possible case in this document, arming community members with an understanding of the reason why we do what we do seems like an important goal.